examples/seq2seq/run_seq2seq.py

```diff
@@ -406,6 +444,14 @@ def preprocess_function(examples):
             ]
 
         model_inputs["labels"] = labels["input_ids"]
+
+        # prepare decoder_input_ids
+        if hasattr(model, "prepare_decoder_input_ids_from_labels"):
+            decoder_input_ids = model.prepare_decoder_input_ids_from_labels(
+                labels=model_inputs["labels"], pad_token_id=tokenizer.pad_token_id
```

Comment:
Why pass the `tokenizer.pad_token_id` here? Are there cases where a seq2seq model does not have a `pad_token_id` ? Or where `tokenizer.pad_token_id` could be different to `model.config.pad_token_id`?